orchagent/orch.cpp

```diff
@@ -172,9 +172,10 @@ size_t Consumer::refillToSync(Table* table)
     return addToSync(entries);
 }
 
-size_t Consumer::refillToSync()
+size_t ConsumerBase::refillToSync()
 {
     auto subTable = dynamic_cast<SubscriberStateTable *>(getSelectable());
+    auto consumertable = dynamic_cast<ConsumerTableBase *>(getSelectable());
```

Comment:
Suggest move this to line 192 to avoid unnecessary convert, also because first if block returned, so line 192 can change from 'else if' to 'if'

Reply:
Done